package.json

```diff
@@ -154,4 +154,4 @@
       "webtorrent.chromeapp.js"
     ]
   }
-}
```

Comment:
I think that we should leave this newline at the end 😉

Reply:
Bizarre! No clue why that happened. Thanks!